modules/exploits/linux/http/pulse_secure_file_disclosure.rb

```diff
@@ -0,0 +1,73 @@
+# Quick hackish exploit for the CVE-2019-11510
```

Comment:
Presuming that you wish to license this module for use in Metasploit, please use the standard comment header here.